tailwind.config.cjs

```diff
@@ -24,6 +24,12 @@ module.exports = {
         osOrange: "#ED5F00",
         lightOrange: "#FFCCA7",
         lightSlate:"#11181C",
+        lightSlate11:"#687076",
```

Comment:
the color codes could use better naming but LGTM! 👍

Reply:
Can you let me know how could I use better naming for variables?

Reply:
Might be worth getting adding the variables we are using in the insights repo (currently private). https://github.com/open-sauced/hot/issues/275

Reply:
I'm thinking of connecting the variables to the current components/icons being treated, examples: `iconPullGrey`, `iconIssueFaded`, `hotIssueIconGrey`, etc